src/DataGridList/DataGridList.js

```diff
@@ -80,7 +80,7 @@ var DataGridList = module.exports = kind(
 	* @method
 	* @private
 	*/
-	constructor: kind.inherit(function (sup) {
+	create: kind.inherit(function (sup) {
```

Comment:
I realize it was part of the original code, but we should get rid of the `return` on L92.